torchtune/modules/transformer.py

```diff
@@ -389,6 +393,7 @@ def __init__(
         self.head_dim = head_dim
         self.causal_mask = None
         self.num_output_chunks = 0
+        self.skip_output_layer = False
```

Comment:
I would prefer the variable `skip_output_projection` to make it even more clear that this is just skipping the normal output projection and nothing else.

Reply:
Would need to update other methods and varaibles as well.